src/utils.ts

```diff
@@ -228,7 +228,7 @@ export function escapeRegExp(string: string): string {
  * @throws Error if the substitution is invalid or if the capture group does not exist.
  */
 export function substituteMatch(match: RegExpExecArray, replacement: string): string {
-    const rx = /\$<.+?>|\${.+?}|\$\d+|\$./g;
+    const rx = /\$<.+?>|\${.+?}|\$\d+|\$~/g;
```

Comment:
This was probably an oversight while replacing all occurrences.

```suggestion
    const rx = /\$<.+?>|\${.+?}|\$\d+|\$./g;
```